cli/tests/tobs-cli-tests/main_test.go

```diff
@@ -42,7 +42,7 @@ func TestMain(m *testing.M) {
 	installObs()
 
 	// FIXME(paulfantom): This should be converted into polling for instances
-	time.Sleep(3 * time.Minute)
+	time.Sleep(5 * time.Minute)
```

Comment:
Since we are planning to refactor those tests and improve them all together I opted for a quick and dirty solution.